src/core/src/core_logic/ConfigurePatchingProcessor.py

```diff
@@ -140,9 +140,7 @@ def __report_consolidated_configure_patch_status(self, status=Constants.STATUS_T
             self.status_handler.add_error_to_status(error_msg, Constants.PatchOperationErrorCodes.DEFAULT_ERROR, current_operation_override_for_error=Constants.CONFIGURE_PATCHING_AUTO_ASSESSMENT)
 
         # write consolidated status
-        self.status_handler.set_configure_patching_substatus_json(status=status,
-                                                                  automatic_os_patch_state=self.current_auto_os_patch_state,
-                                                                  auto_assessment_state=self.current_auto_assessment_state)
+        self.status_handler.set_configure_patching_substatus_json(status=status, automatic_os_patch_state=self.current_auto_os_patch_state, auto_assessment_state=self.current_auto_assessment_state)
```

Comment:
Marking all changes not related to this PR goal. See my other comment on what can be done with these

Reply:
this must be accident file format, i've revert the changes

Reply:
the latest commit took care this unrelated change. done

Reply:
@rane-rajasi Please resolve if satisfied.